resources/lang/en/authorization.php

```diff
@@ -84,6 +84,11 @@
                 'not_owner' => 'Only poster can edit the post.',
                 'topic_locked' => 'Can not edit post of a locked topic.',
             ],
+
+            'store' => [
+                'play_more' => 'Try playing the game before posting on the forums, please! If you have a problem with playing, please post to the Help and Support forum.',
+                'too_many_help_posts' => 'Try playing the game before posting further. If you still have trouble playing, check help pages.',
```

Comment:
Maybe something along the lines of "You need to play the game more before you can make additional posts."?